apps/backend/src/db.ts

```diff
@@ -193,12 +193,16 @@ export const upsertWorkout = async (
     'Something went wrong while upserting workout'
   >
 > => {
+  console.debug(`[db.upsertWorkout] user ${userId} tries to upsert workout`);
   try {
     /* TODO: Fix this to first upsert the workout and its parts, and then delete the potentially unused workout parts. */
     const workoutInDB = await prisma.workout.findFirst({
       where: { id: workoutId },
     });
-    if (!(workoutInDB?.userId === userId)) {
```

Comment:
This prevented users creating new workouts 🥸 It's probably about time we introduce some tests 🐛